packages/ketcher-react/src/script/ui/state/hotkeys.ts

```diff
@@ -42,9 +42,9 @@ export function initKeydownListener(element) {
   }
 }
 
-function removeNotRenderedStruct(actionTool, event, dispatch) {
-  const { code, metaKey } = event
-  if (actionTool.tool === 'paste' && code === 'KeyS' && metaKey) {
+function removeNotRenderedStruct(actionTool, group, dispatch) {
+  const effectedTools = ['paste', 'template']
```

Comment:
Guess, you've meant that
```suggestion
  const affectedTools = ['paste', 'template']
```

Reply:
Thank you. Updated.